src/Sai2Model.h

```diff
@@ -755,6 +761,58 @@ class Sai2Model {
 	 */
 	void displayJoints();
 	void displayLinks();
+	
+	/**
+	 * @brief Computes the joint selection matrix for the joints leading up to the link
+	 * 
+	 * @param link_name 
+	 * @return MatrixXd 
+	 */
+	MatrixXd linkDependency(const std::string& link_name);
+
+	/**
+	 * @brief   Computes ABA
+	 * 
+	 */
+	void forwardDynamics(VectorXd& ddq, const VectorXd& tau);
```

Comment:
Make more explicit comment, and return ddq instead of passing it as parameter. The signature of the function should be
VectorXd forwardDynamics(const VectorXd& tau)

Reply:
Done